### PR TITLE
Tests with maven

### PR DESCRIPTION
**Pull request by nd0761**
Create function to find time slot for requested meeting.

### CODE REVIEW

**Comment by nd0761** [comment]
I don't know why there is so many commits, only last to have anything new(

**Comment by conradgrobler** [comment]
> I don't know why there is so many commits, only last two have anything new(

I think you created this branch from another local branch rather than `master`. As a general rule, you should delete local branches once you have merged the PR and deleted the server branch. You can also use `git remote prune origin` to remove references to remote branches that have been deleted. It is usually simpler to manage everything when you create new branches from master (e.g. `git checkout master` before creating the new branch) unless you have a very specific requirement that the new branch is based on another branch that is not yet merged.

Try to rebase from master (`git rebase master`), fix any merge issues, and then force update the remote branch (`git push origin TestsWithMaven --force`).

**Comment by conradgrobler** [comment]
It could also be that you have not updated you local master branch to match the server, so perhaps try:

- `git checkout master`
- `git pull origin master`
- `git chekout TestsWithMaven`
- `git rebase master`
- fix merge conflicts if there are any and if so also run `git rebase --continue`
- `git push origin TestsWithMaven --force`

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -75,7 +74,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       } else {
         numberOfComments -= 1;
       }
-
+
```
Comment (ipetr0v) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:77]: Try to enable automatic trailing spaces' removal in your IDE

**Comment by nd0761** [comment]
I managed to properly rebased master! Should I create new branch or left that one as is?

**Comment by conradgrobler** [comment]
> I managed to properly rebased master! Should I create new branch or left that one as is?

You can just force push the changes to `origin`, which would hopefully resolve the merge conflict.

**Comment by nd0761** [comment]
I tried to resolve everything by hands but nothing changed..... i don't understand whant happened(((

**Comment by nd0761** [comment]
And none of conflicts disappear(

**Comment by nd0761** [comment]
oh.... or i should commit....

**Comment by nd0761** [comment]
oh(( now i can't push, i receive:

To https://github.com/nd0761/step.git
 ! [rejected]        TestsWithMaven -> TestsWithMaven (non-fast-forward)
error: failed to push some refs to 'https://github.com/nd0761/step.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by nd0761** [comment]
Sorry, i failed it( I created new branch i think it will be a. lot more easily(((((

**Pull request closed without merging**